### PR TITLE
Add linked posts and parent_id vote count

### PR DESCRIPTION
**Pull request by b2ddg**
Add post-links information for StackOverflow data as well as vote counts for the parent post.
`<field name="meta">{"creation_date":"2016-02-28T18:04:22.273","accepted":"0","post_links":"40924,3294","parent_score","15"}</field>`

@zachthompson

### CODE REVIEW

File: lib/longtail/stackoverflow/posts.pl [hunk 1]
```diff
@@ -61,7 +61,31 @@
     }
 
     print qq(\nTYPE: $name\t$data->{src_domain}\n);
-    
+
+    my %post_links = ();
+    if (-e "$stack_dir/$data->{src_domain}/PostLinks.xml") {
```
Comment (zachthompson) [inline lib/longtail/stackoverflow/posts.pl]: If this doesn't exist, don't we want do more than skip it?  We should probably be better about `die`ing if files don't exist that we expect.

Comment (b2ddg) [inline lib/longtail/stackoverflow/posts.pl]: I think we want to process the files even if PostLinks.xml isn't found. I think the post links can be classified as non-essential information to the IA itself.

Comment (zachthompson) [inline lib/longtail/stackoverflow/posts.pl]: Gotcha.

File: lib/longtail/stackoverflow/README [hunk 2]
```diff
@@ -14,8 +14,9 @@ rm *.7z*
 rm -rf meta*
 find . | egrep -e 'Badges|PostHistory|Comments|PostLinks' | xargs rm -rf
```
Comment (zachthompson) [inline lib/longtail/stackoverflow/README]: Should remove `PostLinks` from here.

Comment (b2ddg) [inline lib/longtail/stackoverflow/README]: good call

**Comment by zachthompson** [comment]
Thank you.

**Pull request merged**